<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoClass</class>
 <widget class="QWidget" name="ToDoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToDo</string>
  </property>
  <widget class="QLineEdit" name="lineEditInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetTasks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOnTop">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>置顶</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ToDo.qrc"/>
 </resources>
 <connections/>
</ui>
